api/routes/nsadm.go

```diff
@@ -4,9 +4,11 @@ import (
 	"net/http"
 	"strconv"
 
+	"github.com/asaskevich/govalidator"
```

Comment:
Please add an utility inside the pkg/validator so we avoid adding the extra package here.